client/src/test/scala/dwolla/cloudflare/RateLimitClientSpec.scala

```diff
@@ -643,6 +655,34 @@ class RateLimitClientSpec extends Specification {
           |}
         """.stripMargin
 
+      val invalidRateLimit =
+        """{
```

Comment:
These JSON strings are a good opportunity to use the JSON literal feature of circe, to make sure this is valid JSON being returned.

Reply:
I like this, but punting on it for now.